index.bs

```diff
@@ -1307,6 +1313,29 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Preference enumeration (enum <dfn enum>AttestationPreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use the {{AttestationPreference}} enum to specify their preference regarding attestation types used 
+during credential generation.
+
+<pre class="idl">
+    enum AttestationPreference {
+        "low-cost",
+        "verifiable" 
+    };
+</pre>
+
+<ul>
+    <li><dfn>low-cost</dfn> - indicates that the [=[RP]=] prefers a low-cost attestation type. For example, if a user agent
+      supports self attestion and Privacy CA, the user agent SHOULD use self attestation, avoiding calls to the Privacy
```

Comment:
a) it is not the user agent supporting attestation but the authenticator.
b) we should use more general language, i.e.:
if an authenticator supports self attestation and some ther attestation type, the authenticator SHOULD use self attestation.

Reply:
Rewrote this. Can you take another look?